jms/1.7/modules/ROOT/pages/jms-performance.adoc

```diff
@@ -1,47 +1,71 @@
-= JMS Tuning For Performance
+= Tune JMS Connector for Performance
 :keywords: jms, connector, consume, message, source, listener
 :page-aliases: connectors::jms/jms-performance.adoc
 
+Performance can be a factor when processing the loading of messages. The following recommendations can help you get the most of Anypoint Connector for JMS (JMS Connector) for your Mule applications.
 
+== Do not Disable Connections Caching
```

Comment:
```suggestion
== Do Not Disable Connections Caching
```
or
```suggestion
== Don't Disable Connections Caching
```

Reply:
But...we typically do not want to make titles complete sentences, which these are.
I first thought we could just use a bullet list, but these are too long for that.
Maybe "Connection Caching Reuse"?